docs/docs/getting-started/first-steps.md

```diff
@@ -40,45 +40,61 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 
     ```bash
-    RESOURCE_GROUP=constellation # enter name of new resource group for your cluster here
-    LOCATION=westus # enter location of resources here
-    SUBSCRIPTION_ID=$(az account show --query id --out tsv)
-    SERVICE_PRINCIPAL_NAME=constell
-    az group create --name "${RESOURCE_GROUP}" --location "${LOCATION}"
-    az group create --name "${RESOURCE_GROUP}-identity" --location "${LOCATION}"
-    az ad sp create-for-rbac -n "${SERVICE_PRINCIPAL_NAME}" --role Owner --scopes "/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP}" | tee azureServiceAccountKey.json
-    az identity create -g "${RESOURCE_GROUP}-identity" -n "${SERVICE_PRINCIPAL_NAME}"
-    identityID=$(az identity show -n "${SERVICE_PRINCIPAL_NAME}" -g "${RESOURCE_GROUP}-identity" --query principalId --out tsv)
-    az role assignment create --assignee-principal-type ServicePrincipal --assignee-object-id "${identityID}" --role 'Virtual Machine Contributor' --scope "/subscriptions/${SUBSCRIPTION_ID}"
-    az role assignment create --assignee-principal-type ServicePrincipal --assignee-object-id "${identityID}" --role 'Application Insights Component Contributor' --scope "/subscriptions/${SUBSCRIPTION_ID}"
-    echo "subscription: ${SUBSCRIPTION_ID}
-    tenant: $(az account show --query tenantId -o tsv)
-    location: ${LOCATION}
-    resourceGroup: ${RESOURCE_GROUP}
-    userAssignedIdentity: $(az identity show -n "${SERVICE_PRINCIPAL_NAME}" -g "${RESOURCE_GROUP}-identity" --query id --out tsv)
-    appClientID: $(jq -r '.appId' azureServiceAccountKey.json)
-    clientSecretValue: $(jq -r '.password' azureServiceAccountKey.json)"
+    constellation iam create azure --region=westus --resourceGroup=constellTest --servicePrincipal=spTest
     ```
 
-    Fill the values produced by the script into your configuration file.
+    Find an exact command description in our [CLI reference](https://docs.edgeless.systems/constellation/reference/cli).
 
-    :::tip
+    By default, Constellation uses `Standard_DC4as_v5` CVMs (4 vCPUs, 16 GB RAM) to create your cluster. Optionally, you can switch to a different VM type by modifying **instanceType** in the configuration file. For CVMs, any VM type with a minimum of 4 vCPUs from the [DCasv5 & DCadsv5](https://docs.microsoft.com/en-us/azure/virtual-machines/dcasv5-dcadsv5-series) or [ECasv5 & ECadsv5](https://docs.microsoft.com/en-us/azure/virtual-machines/ecasv5-ecadsv5-series) families is supported.
 
-    Alternatively, you can leave `clientSecretValue` empty and provide the secret via the `CONSTELL_AZURE_CLIENT_SECRET_VALUE` environment variable.
+    You can also run `constellation config instance-types` to get the list of all supported options.
 
-    :::
+    Paste the output into the corresponding fields of the `constellation-conf.yaml` file created in the previous step.
+    </tabItem>
```

Comment:
I think this sentence belongs just below the bash command

Reply:
I changed that. I also noticed we didn't mention `constellation config instance-types` in every CP's tab. Should we do so? On that, I also noticed we don't mention supported AWS machines anywhere. I think we should add them somewhere, right?

Reply:
Yes, we should do both. AWS types are probably too much to list them all, but we should at least link to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enable-nitrotpm-prerequisites.html

Reply:
Added that [here](https://github.com/edgelesssys/constellation/pull/711/commits/d71ccef4ffd52869affcaa61a24af3a80ee9d0f0). Could you verify it for technical correctness?

Reply:
That's fine 👍